website/screens/components/dropdown/usage/DropdownUsagePage.tsx

```diff
@@ -0,0 +1,83 @@
+import { DxcList, DxcStack, DxcText } from "@dxc-technology/halstack-react";
+import DocFooter from "../../../common/DocFooter";
+import HeadingLink from "../../../common/HeadingLink";
+import Image from "@/common/Image";
+import Figure from "../../../common/Figure";
+import dropdownIconUsage from "./images/dropdown_icon_usage.png";
+
+const DropdownUsagePage = () => {
+  return (
+    <DxcStack gutter="xxxlarge">
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Usage</HeadingLink>
+        <HeadingLink level={3}>Appearance</HeadingLink>
```

Comment:
The section appearance should be wrapped by a DxcStack level 3.

Reply:
The actual documentation of the component doesn't have a _Usage_ section so I couldn't put anything inside. The other option would be to do this (which I found a bit lacking): 
```
      <DxcStack gutter="large">
        <HeadingLink level={2}>Usage</HeadingLink>
      </DxcStack>
```